apps/openmw/mwmechanics/aicombat.cpp

```diff
@@ -221,12 +221,12 @@ namespace MWMechanics
 
         if(movement.mRotation[2] != 0)
         {
-            if(zTurn(actor, osg::DegreesToRadians(movement.mRotation[2]))) movement.mRotation[2] = 0;
+            if(zTurn(actor, movement.mRotation[2])) movement.mRotation[2] = 0;
```

Comment:
To be clear, I mean the MWMechanics::Movement struct. We should use it to store rotations in Radians <i>or</i> Degrees, but not both.